torchaudio/backend/sox_io_backend.py

```diff
@@ -20,22 +21,24 @@ def __init__(self, sample_rate: int, num_frames: int, num_channels: int):
 
 
 @_mod_utils.requires_module('torchaudio._torchaudio')
-def info(filepath: str) -> AudioMetaData:
+def info(filepath: Union[str, Path]) -> AudioMetaData:
     """Get signal information of an audio file.
 
     Args:
-        filepath (str): Path to audio file
+        filepath (str/pathlib.Path): Path to audio file
```

Comment:
Could you add note on the `pathlib.Path` compatibility, like the following one? (same for the other functions too.)

```suggestion
        filepath (str or pathlib.Path): Path to audio file. Note that the function is annotated only as ``str`` for the reason TorchScript compiler compatibility, this function can also handle `pathlib.Path`.
```

Reply:
How about the following?
```
This function also handles `pathlib.Path` outside of TorchScript.
```

Since we'd be making an official statement here, should we also update soundfile too? And should we add some tests to guarantee this?

Reply:
That all makes sense - good to know, thanks!
I'll update this tonight with the suggestions you added.

Reply:
@vincentqb `soundfile_backend` (and the old `sox_backend`) use the docstring from `common.py`, so I added the note regarding pathlib there too now.